CAN_Logger_with_Autobaud_and_Requests/CAN_Logger_with_Autobaud_and_Requests.ino

```diff
@@ -926,7 +926,7 @@ void loop(void) {
       if (send_request_timer > REQUEST_TIMING){
         send_request_timer = 0;
         txmsg.len = 3;
-        txmsg.id = 0x18EA00F9;
+        txmsg.id = 0x18EAFFF9;
```

Comment:
this is a good example of a place where extracting this 'magic number' into a constant would have resulted in a single line change. 

Please consider defining all magic numbers as well-named constants and then using only the well named constants in  imperative statements.